source/developers/plugins/blueprints.rst

```diff
@@ -94,7 +94,7 @@ Here are some best practices to help you:
             </paths>
 
        * You can also use this to enforce single page blueprints by using ``<excludes> <pattern>^/.*</pattern> </excludes>`` in your page type's config.xml, effectively forbidding from creating a new page.
-    * Ensure your blueprint supports :ref:`in-context-editing`.
+    * Ensure your blueprint supports :ref:`experience-builder`.
     * For most sites, you'll have to override Studio's default navigation menu tags. You can do this by reading :ref:`templating-rendering-navigation`.
```

Comment:
I think this is no longer the case. Not entirely sure what this refers to though, but devs should be able to just use the nav macros.

Reply:
@alhambrav what's the answer to this?

Reply:
We had a discussion on slack and @rart is going to review it to help me adjust the docs.  Thank you.